packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -5,19 +5,33 @@ import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 import 'package:meta/meta.dart';
 import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
 
-import 'package:platform_detect/platform_detect.dart' show browser;
-
 const _safariTargetTopSchemes = {
   'mailto',
   'tel',
   'sms',
 };
+String _getUrlScheme(String url) => Uri.tryParse(url)?.scheme;
+
+bool _isSafariTargetTopScheme(String url) =>
+    _safariTargetTopSchemes.contains(_getUrlScheme(url));
+
+// Taken from package:platform_detect.
+bool _navigatorIsSafari(html.Navigator navigator) {
```

Comment:
nice - not worth a 3P dependency for sure.

Reply:
Yeah, we suspected that we'd need this for more browsers, but it turns out that we didn't :P